backend/src/routes/api/k8s/pass-through.ts

```diff
@@ -33,26 +53,22 @@ const setupRequest = async (
     safeURL = queryParams.join('?');
   }
 
-  const requestOptions = await getDirectCallOptions(fastify, request, url);
-
-  return {
+  const updatedData = {
```

Comment:
```suggestion
  const updatedData: PassThroughData = {
```

Reply:
@andrewballantyne This is the backend lint settings which we didn't change and agreed it doesn't worth the effort to change for now because we will refactor it one day.

Reply:
Pass through API is not going anywhere -- this won't just be removed some day, this was a gap. But the way TS infers types, it should not cause an issue.